medical_pathology/models/medical_pathology.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from openerp import _, api, fields, models
+from openerp.exceptions import ValidationError
+
+
+class MedicalPathology(models.Model):
+    _name = 'medical.pathology'
+    _description = 'Medical Pathology'
+
+    name = fields.Char(
+        required=True,
+        translate=True,
+    )
+    code = fields.Char(
+        required=True,
+    )
+    code_type_id = fields.Many2one(
+        string='Code Type',
+        comodel_name='medical.pathology.code.type',
+        index=True,
+    )
+    notes = fields.Text(
+        translate=True,
+    )
+    category_id = fields.Many2one(
```

Comment:
Note: this is so that a pathology can be related to all of its different code types (ICD-9, 10, SNOMED).

There is one hole here though - each of the code types contains slightly different naming and descriptions, while still being related to the same pathology. I am not quite sure how to circumvent this & will be adding it to v11 roadmap